### ISSUE TITLE
Looking for a security contact to disclose a vulnerability

### ISSUE THREAD

**Issue by paul-gerste-sonarsource** [lib/pq#1153]
Hi! 👋

I found and verified a security vulnerability affecting the latest version of pq.

I want to disclose the issues to the maintainers responsibly, so I'm looking for a security contact (e.g., an email address) to send the details privately. Also happy to help with patches if needed.

Thanks!

**Comment by paul-gerste-sonarsource** [comment]
Hey @rafiss @johto, you were the latest active maintainers I could find in PRs. Would one of you be able to give this a look? If not, who would be a better person to contact?

Thanks!

**Comment by hmh** [comment]
Is this the same as in #1141  ?

**Comment by paul-gerste-sonarsource** [comment]
No, it's something different. A few months after opening this issue, I created https://github.com/lib/pq/pull/1161, which contains more information on the bug. Let me know if it's unclear or if you have any questions!
